tools/pot/openvino/tools/pot/algorithms/quantization/bias_correction/algorithm.py

```diff
@@ -283,8 +283,11 @@ def _create_results_after_nodes(self, output_nodes):
         outputs_data = []
         for output_node in output_nodes:
             output_name = nu.create_node_name(output_node, str)
-            result_name = output_name + '/result'
+            result_name = output_name + '/sink_port_0'
```

Comment:
@KodiaqQ, are we breaking compatibility with IRv10 with this change? Can you take a look at the changes and highlight what can break it?